File: remove_repeat.py
```Remove duplicate characters in a given string keeping only the first occurrences. For example, if the input is ‘tree traversal’ the output will be "tre avsl".```

def get_unique(str):
  arr = list(str)
  print(arr)
  unique = []
  for i in arr:
    if i not in unique:
      unique.append(i)
  print(''.join(unique))

get_unique('tree traversal')

File: remove_repeat.js
//Remove duplicate characters in a given string keeping only the first occurrences. For example, if the input is ‘tree traversal’ the output will be "tre avsl".

function removeDupedChars(str){
  let arr, uniqueChars

  arr = str.split('')

  uniqueChars = []

  for (let i = 0; i < arr.length; i++){

    if (!uniqueChars.includes(arr[i])){
      uniqueChars.push(arr[i])
    }

  }

  return uniqueChars.join('')
}

removeDupedChars('tree traversal')
